hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/GenericRecordQueryReader.java

```diff
@@ -244,35 +244,35 @@ private Object readIndexed(InternalGenericRecord record, String path, int index)
         FieldType type = record.getFieldType(path);
         switch (type) {
             case BYTE_ARRAY:
-                return record.readByteFromArray(path, index);
+                return record.getByteFromArray(path, index);
```

Comment:
do we really need to write `FromArray` int the method name? is the method signature not telling and is this not clear from method signature? I personally prefer shorter names where possible and let the method signature describe it. If it is not very obvious from the signature, then longer method name can be used.

Reply:
@ihsandemir I am skipping this internal name suggestions since it is not relevant to this pr to merge it as soon as possible.